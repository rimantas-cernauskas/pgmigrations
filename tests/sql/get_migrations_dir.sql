-- Testing function pgmigrations.get_migrations_dir()
\i tests/sql/setup/init.sql

BEGIN;
SELECT plan(1);

SET ROLE __pgmigrations_testuser;

SELECT is(
     pgmigrations.get_migrations_dir()
    ,'migrations/contrib_regression'
    ,'should get database specific migrations directory'
);

SELECT * FROM finish();
ROLLBACK;