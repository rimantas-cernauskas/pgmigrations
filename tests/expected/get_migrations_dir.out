-- Testing function pgmigrations.get_migrations_dir()
\i tests/sql/setup/init.sql
\set ECHO errors
1..1
ok 1 - should get database specific migrations directory
